torch/_torch_docs.py

```diff
@@ -3718,13 +3718,15 @@ def merge_dicts(*dicts):
 
 add_docstr(torch.eye,
            r"""
-eye(n, m=None, *, out=None, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
+eye(n, m=None, k=None, *, out=None, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
```

Comment:
`k` is a keyword-only argument in the Array API with default value 0.
```suggestion
eye(n, m=None, *, k=0, out=None, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
```